### DISCUSSION TITLE
New queue system fails on Laravel Vapor

### DISCUSSION THREAD

**Discussion by dbfx** [spatie/laravel-slack-alerts#53] [Q&A]
Hi all

Just reporting an issue that may be a roadblock for others, possibly solved in docs or in code, but with Laravel Vapor the recent queue change was breaking. 

It assumes "default" is a functional queue, but with vapor the queue (if unspecified) is {app_name}-{environment}. 

Thank you

**Comment by mikelmao** [comment]
Also facing this issue

**Comment by 1stevengrant** [comment]
also seeing this issue

**Comment by 1stevengrant** [comment]
The fix for us was just to add an `env` variable `SLACK_ALERT_QUEUE=vaporprojectname-prod` and redeploy.
